flow/instrumentation.h

```diff
@@ -14,6 +14,8 @@
 
 namespace flow {
 
+// DEPRECATED
+// The frame per second FPS could be different than 60 (e.g., 120).
 static const double kOneFrameMS = 1e3 / 60.0;
```

Comment:
are there consumers that still exist that use this?

Reply:
It's used for drawing 3 horizontal lines: 16ms, 32ms, 48ms (https://github.com/flutter/engine/blob/master/flow/instrumentation.cc#L92)

Eventually, we should remove this altogether and get the refresh rate from https://github.com/flutter/engine/pull/7002 

I feel that it's better to do it in another PR to keep each one small and focused.